events.md

```diff
@@ -1,9 +1,18 @@
 ---
 layout: page
 title: Events
-image: https://images.unsplash.com/photo-1535016120720-40c646be5580?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80
+description: The hub of all OLS events
+image: https://images.unsplash.com/photo-1503428593586-e225b39bddfe
 photos:
-  name: Alex Litvin
-  url: https://unsplash.com/photos/MAYsdoYpGuk
+  name: The Climate Reality Project
+  license: CC BY-SA 4.0
+  url: https://images.unsplash.com/photo-1503428593586-e225b39bddfe
 ---
 
+## Upcoming Events
+
+| Date/Time | Topic/Event | Venue/Location | Speakers/Contact | Organisers |
+|-----------|-------------|----------------|------------------|------------|
+{%- for event in site.data.events -%}
+| {{ event.date }} at {{ event.time }} | {{ event.title }} | - | {{ event.people }} | - |
```

Comment:
What about event organizer? It should be a metadata

Reply:
`event.people` is not in the `event` metadata. It could be better to have in the event metadata a list with contact people that could be put there

Reply:
There is no link here to the event page or the external link

Reply:
Done. 👍